### ISSUE TITLE
Wallet command: implement wallet list or wallet ls

### ISSUE THREAD

**Issue by dan-da** [maidsafe/safe-api#391]
Presently I cannot find a way to list wallets, or to determine which is the default wallet.

I think that `wallet list` or `wallet ls` would solve this.

**Comment by bochaco** [comment]
Yes this is something missing, but we depend on the ability to keep track of things created with an account, this used to be done by apps storing such data into the "default containers", which every account has, but we are now trying to have the indexing and labeling of data (https://safenetforum.org/t/rfc-labelled-data-indexing-and-token-authorisation/30597/62), to be tracked by issue #129 .

Once we have that capability you will be able to retrieve all type of data you created, even using URLs like `safe:///<labels set to wallets>` (issue #357).

**Comment by dan-da** [comment]
ok, cool.  looking forward to that...
